src/modules/conditions.js

```diff
@@ -220,6 +225,15 @@ export const getConditions = ({
             value = parts[1]
         }
 
+        if (
+            [VALUE_TYPE_TIME, VALUE_TYPE_DATETIME].includes(dimension.valueType)
+        ) {
+            value = value.replaceAll(API_TIME_DIVIDER, UI_TIME_DIVIDER)
```

Comment:
Should this be done with some kind of date/time formatting library instead?

Reply:
Assuming there is no edge-case with the backend format where the replace does not do the right thing, and assuming we want a fixed format regardless of user preference and locale, then this solution works fine.

Reply:
Actually we figured out that the `datetime` conditions doesn't work, https://jira.dhis2.org/browse/DHIS2-13114, so I assume we'll have to adjust this at some point once the backend has a new format, but that'll be a separate PR later.